src/components/Footer/Footer.js

```diff
@@ -101,24 +111,28 @@ const Footer = () => {
 
   return (
     <Row className="footer-row">
-      <Col lg={4} xs={8} className="col-logo">
+      <Col lg={4} xs={6} className="col-logo">
         <img src={logo} className="footer-logo" alt="logo" />
```

Comment:
The logo looks cropped:
![image](https://user-images.githubusercontent.com/25405578/107127357-2e2e8980-687b-11eb-83f3-c08116f3a5dd.png)

You can change the width in CSS, and eliminate the height limit.